### PR TITLE
Various bug fixes for base images

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: python/generate-dockerfile.sh [hunk 1]
```diff
@@ -38,6 +39,7 @@ append_setup_script() {
 # $1: base version
 # $2: path
 # $3: variants
+# S4: distro
```
Comment (Page-) [inline python/generate-dockerfile.sh]: This should be `$4`?

File: python/generate-dockerfile.sh [hunk 2]
```diff
@@ -48,9 +50,10 @@ set_pythonpath() {
 		fi
 
 		if [ $1 != "2.7" ]; then
-			echo "ENV PYTHONPATH $PYTHON3_PATH" >> $tmp_path/Dockerfile
+			PPATH="PYTHON3_PATH_$4"
+			echo "ENV PYTHONPATH $(eval echo \$$PPATH):\$PYTHONPATH" >> $tmp_path/Dockerfile
```
Comment (Page-) [inline python/generate-dockerfile.sh]: Can't this just be `echo "ENV PYTHONPATH $PPATH:\$PYTHONPATH" >> $tmp_path/Dockerfile`?

File: python/generate-dockerfile.sh (hunk 1 above)
Comment (nghiant2710) [inline python/generate-dockerfile.sh]: Updated!

File: python/generate-dockerfile.sh (hunk 2 above)
Comment (nghiant2710) [inline python/generate-dockerfile.sh]: This must be the value of string variable (value of PPATH) not just PPATH

Comment (Page-) [inline python/generate-dockerfile.sh]: Ah, I get it, in that case you can use `echo "ENV PYTHONPATH ${!PPATH}:\$PYTHONPATH" >> $tmp_path/Dockerfile`

Comment (nghiant2710) [inline python/generate-dockerfile.sh]: Updated!

**Comment by Page-** [comment]
SGTM

**Pull request merged**